RecoMuon/MuonIdentification/src/MuonShowerInformationFiller.cc

```diff
@@ -276,36 +276,7 @@ MuonShowerInformationFiller::hitsFromSegments(const GeomDet* geomDet,
 //
 // Find cluster
 //
-MuonTransientTrackingRecHit::MuonRecHitContainer 
-MuonShowerInformationFiller::findPhiCluster(MuonTransientTrackingRecHit::MuonRecHitContainer& muonRecHits, 
-                              const GlobalPoint& refpoint) const {
-
-  if ( muonRecHits.empty() ) return muonRecHits;
-
-  //clustering step by phi
-  float step = 0.05;
-  MuonTransientTrackingRecHit::MuonRecHitContainer result;
-
-  stable_sort(muonRecHits.begin(), muonRecHits.end(), AbsLessDPhi(refpoint));
 
-  for (MuonTransientTrackingRecHit::MuonRecHitContainer::const_iterator ihit = muonRecHits.begin(); ihit != muonRecHits.end() - 1; ++ihit) {
-      if (fabs(deltaPhi((*(ihit+1))->globalPosition().phi(), (*ihit)->globalPosition().phi() )) < step) {

```

Comment:
Hmm.. I restarted from the original code to pass vector<GlobalPoint> but it did not solve the problem. I would assume that sorting in the loop is really a problem.